src/picker/rofimoji.py

```diff
@@ -101,10 +101,17 @@ def parse_arguments(self) -> argparse.Namespace:
         )
         parser.add_argument(
             '--rofi-args',
-            dest='rofi_args',
+            dest='selector_args',
```

Comment:
I guess you left this in to be backwards-compatible. But do you know what happens if I set both `--rofi-args` and `--selector-args`? Are they appended?

Reply:
Yeah, backwards compatibility is my goal there, but I didn't think about that particular situation. I'll try to go back and test that today to see how it works.

Reply:
Thanks. But for the record, I am not averse to breaking changes, if reasonable. But even better would be a more graceful upgrade path with backwards compatibility, like you're trying to achieve 🙂